rmgpy/solver/base.pyx

```diff
@@ -181,8 +181,8 @@ cdef class ReactionSystem(DASx):
         return (self.__class__, (self.termination,))
     
      
-    cpdef initializeModel(self, list coreSpecies, list coreReactions, list edgeSpecies, list edgeReactions, list surfaceSpecies,
-                          list surfaceReactions, list pdepNetworks=None, atol=1e-16, rtol=1e-8, sensitivity=False, 
+    cpdef initializeModel(self, list coreSpecies, list coreReactions, list edgeSpecies, list edgeReactions, list surfaceSpecies=[],
```

Comment:
I think this may be OK here, and may help the cythonization, but I'm always wary of using mutable objects like lists as default variables (eg https://github.com/ReactionMechanismGenerator/RMG-Py/issues/640). But I don't have time to delve deep here. Can you reassure us this usage is safe?